raiden-dapp/src/locales/en.json

```diff
@@ -536,6 +536,7 @@
   "hub-list": {
     "header": "Current top 3 suggested hubs (best first)",
     "error": "Error: List cannot be fetched",
+    "no-results": "Currently no nodes are available. Please try later again or connect to a known node.",
```

Comment:
@christianbrb This working is not perfect, any suggestions?

Reply:
Done, made suggestions

Reply:
Thanks!